components/dbutil/src/main/java/com/wso2telco/dbutils/fileutils/FileReader.java

```diff
@@ -43,29 +43,104 @@
 		try {
 
 			fileStream = new FileInputStream(filePath);
+
 			props.load(fileStream);
+
 			mediatorConfMap.put("ussdGatewayEndpoint", props.getProperty("ussdGatewayEndpoint"));
 			mediatorConfMap.put("hubSubsGatewayEndpoint", props.getProperty("hubSubsGatewayEndpoint"));
 			mediatorConfMap.put("hubGateway", props.getProperty("hubGateway"));
 			mediatorConfMap.put("requestRouterUrl", props.getProperty("requestRouterUrl"));
 			mediatorConfMap.put("sendSMSResourceURL", props.getProperty("sendSMSResourceURL"));
 		} catch (FileNotFoundException e) {
 
-			LOG.debug(PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file not found in " + filePath, e);
+			log.debug(PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file not found in " + filePath, e);
 		} catch (IOException e) {
 
-			LOG.debug("unable to close " + PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file ", e);
+			log.debug("unable to close " + PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file ", e);
 		} finally {
 
 			try {
 
 				fileStream.close();
 			} catch (IOException e) {
 
-				LOG.debug("unable to close " + PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file ", e);
+				log.debug("unable to close " + PropertyFileNames.MEDIATOR_CONF_FILE.getFileName() + " file ", e);
 			}
 		}
 
 		return mediatorConfMap;
 	}
+
```

Comment:
in the utility lib ,there cannt be any business specific logic . pls generalize the file reading.